Loading required package: raster
Loading required package: methods
Loading required package: sp
Warning messages:
1: In `[<-`(`*tmp*`, i, value = raster(rasters1[i])) :
  implicit list embedding of S4 objects is deprecated
2: In `[<-`(`*tmp*`, i, value = raster(rasters1[i])) :
  implicit list embedding of S4 objects is deprecated
3: In `[<-`(`*tmp*`, i, value = raster(rasters1[i])) :
  implicit list embedding of S4 objects is deprecated
4: In `[<-`(`*tmp*`, i, value = raster(rasters1[i])) :
  implicit list embedding of S4 objects is deprecated
5: In `[<-`(`*tmp*`, i, value = raster(rasters1[i])) :
  implicit list embedding of S4 objects is deprecated
6: In `[<-`(`*tmp*`, i, value = raster(rasters1[i])) :
  implicit list embedding of S4 objects is deprecated
7: In `[<-`(`*tmp*`, i, value = raster(rasters1[i])) :
  implicit list embedding of S4 objects is deprecated
Loading required package: raster
Loading required package: methods
Loading required package: sp
Loading required namespace: ncdf4
Loading required package: raster
Loading required package: methods
Loading required package: sp
Error in file(filename, "r", encoding = encoding) : 
  cannot open the connection
Calls: source -> file
In addition: Warning message:
In file(filename, "r", encoding = encoding) :
  cannot open file 'tscale_src.R': No such file or directory
Execution halted
['/home/joel/src/TOPOMAP/toposubv2/topoMAPP', '/usr/local/lib/python2.7/dist-packages/pyandoc-0.0.1-py2.7.egg', '/usr/lib/python2.7', '/usr/lib/python2.7/plat-x86_64-linux-gnu', '/usr/lib/python2.7/lib-tk', '/usr/lib/python2.7/lib-old', '/usr/lib/python2.7/lib-dynload', '/usr/local/lib/python2.7/dist-packages', '/usr/lib/python2.7/dist-packages', '/usr/lib/python2.7/dist-packages/PILcompat', '/usr/lib/python2.7/dist-packages/gtk-2.0', '/usr/lib/python2.7/dist-packages/ubuntu-sso-client', '/home/joel/src/TOPOMAP/toposubv2/topoMAPP/']
code.py
Running:['Rscript', './rsrc/getDEM_points.R', '/home/joel/sim/topomap_test/', '/home/joel/data/DEM/srtm', '0.75', '/home/joel/data/GCOS/points_all.txt', '2', '3']
Running:['Rscript', './rsrc/clipToEra.R', '/home/joel/sim/topomap_test/', '0.75']
Running:['Rscript', './rsrc/tscale_plevel.R', '/home/joel/sim/topomap_test/', '/home/joel/data/GCOS/points_all.txt', '2', '3']
Traceback (most recent call last):
  File "code.py", line 51, in <module>
    shp.main(wd , config['main']['pointsFile'], config['main']['lonCol'], config['main']['latCol'])
  File "/home/joel/src/TOPOMAP/toposubv2/topoMAPP/domain_setup/makeShape.py", line 33, in main
    run_rscript_fileout(path2script,[wd, pointsFile, loncol, latcol])
  File "/home/joel/src/TOPOMAP/toposubv2/topoMAPP/domain_setup/makeShape.py", line 51, in run_rscript_fileout
    subprocess.check_output(cmd)
  File "/usr/lib/python2.7/subprocess.py", line 574, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
subprocess.CalledProcessError: Command '['Rscript', './rsrc/tscale_plevel.R', '/home/joel/sim/topomap_test/', '/home/joel/data/GCOS/points_all.txt', '2', '3']' returned non-zero exit status 1
